packages/core/src/events/handlers/notifyToClientEvaluationResultCreatedJob.ts

```diff
@@ -10,6 +10,8 @@ export const notifyToClientEvaluationResultCreatedJob = async ({
   data: EvaluationResultCreatedEvent
 }) => {
   const { evaluation, documentLog, evaluationResult } = event.data
+  // FIXME: Change this query to include the error
+  // This has to be done with care because it can break aggregations
```

Comment:
Once we have the errors saved into the DB and we see everything is working will show it in the UI, in evaluation results table and document logs table. 

This comment is to send in the realtime the error together with the evaluation result.

I'll do this in another PR.